lib/registrant.rb

```diff
@@ -0,0 +1,25 @@
+class Registrant 
+  attr_reader :name,
+              :age,
+              :permit
+  attr_accessor :license_data
+
+  def initialize(name, age, permit = false, license_data ={:written=>false, :license=>false, :renewed=>false})
```

Comment:
Great use of default arguments! Since this is getting to be on the longer side, I think it would also be fine to instantiate every registrant with false values for the `license_data` hash:
```
def initialize(name, age, permit = false)
  ...
  @license_data = {
    written: false,
    license: false,
    renewed: false
  }
end
```